src/lang/es.json

```diff
@@ -1,12 +1,15 @@
 {
+  "I18N.LANGUAGE": "Español",
```

Comment:
Hey, thanks for the PR. Mostly to understand, what’s the point of these 2 lines? I have seen that being used a couple of times but I don’t understand the reason why people use them.

Reply:
Hi:

I currently maintain more than 100 translations at github and gitlab. Obviously I cant check all of them manually, so I have automated the process of checking if there are new keys. That's why I tag all the repositories I maintain the translation. To keep track of which ones I have to update when I check for missing keys.

Great work! Thanks

Reply:
Would there be another way to do that without introducing additional keys into the language file itself? I am a bit hesitant to do that because that's not the purpose of the language files.

Reply:
Sure, maintaining an external file with the modules I contribute, since there is no other way to see which one I maintain except looking at the contributions of the es.json file of each particular repo.

Anyway, feel free to remove it, no prob, your repo, your rules. Only that I wont be automatically notified when something gets updated, and probably will notice it just by using it in game.